src/AutoMapper/TypeMap.cs

```diff
@@ -105,7 +105,7 @@ public int MaxDepth
 
         public PropertyMap[] GetPropertyMaps()
         {
-            return _sealed ? _orderedPropertyMaps : _propertyMaps.Concat(_inheritedMaps).ToArray();
+            return _orderedPropertyMaps ?? _propertyMaps.Concat(_inheritedMaps).ToArray();

```

Comment:
I'm not sure what you mean, this is about the order of properties in a TypeMap, so how can a PropertyMap constructor help with that? The order field is set, but is not used because sealed is not set to true yet.